fins/client.go

```diff
@@ -21,13 +22,15 @@ type Client struct {
 	src  finsAddress
 	sid  byte
 	closed bool
+	responseTimeout time.Duration
```

Comment:
maybe better to have it as int ? and name responseTimeoutMs ? The problem that i see - user will use: setTimeout( 5* time.Millisecond) and you will multiply once again on time.Millisecond.  Or not to multiply ... What do you think?